source/gfoidl.Stochastics/Statistics/SampleBuilder.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using gfoidl.Stochastics.Builders;
+
+namespace gfoidl.Stochastics.Statistics
+{
+    public class SampleBuilder
+    {
+        private double[] _values;
+        private double _min;
+        private double _max;
+        private double _avg;
+        //---------------------------------------------------------------------
+        public IEnumerable<double> Add(IEnumerable<double> values)
```

Comment:
This method name is not the best -- see https://github.com/gfoidl/Stochastics/pull/22#pullrequestreview-90125578